.circleci/config.yml

```diff
@@ -0,0 +1,79 @@
+## tic Circle CI template: linux-deploy
+## revision date: 2022-08-28
+version: 2.1
+jobs:
+  r-release:
+    machine:
+        image: ubuntu-2204:current
+    resource_class: small
+
+    steps:
+      - checkout
+
+      # create a unique env var for the cache. Unfortunately normal env vars
+      # are not picked up by the cache, therefore this workaround is needed.
+      # See https://discuss.circleci.com/t/cannot-use-circle-yml-environment-variables-in-cache-keys/10994/7
+      - run: echo "$(Rscript -e 'cat(format(getRversion()))')" > /tmp/R_version
+      - run: echo "v1" > /tmp/cache_vers  # Increment this to invalidate cache
+      - restore_cache:
+          key: R-package-library-{{ checksum "/tmp/cache_vers" }}-{{ checksum "/tmp/R_version" }}
+
+      # install deps and check pkg ---------------------------------------------
+      - run:
+          name: "[r-release] Install dependencies"
+          command: |
+            sudo apt update -qq
+            sudo apt install --no-install-recommends software-properties-common dirmngr
+            wget -qO- https://cloud.r-project.org/bin/linux/ubuntu/marutter_pubkey.asc | sudo tee -a /etc/apt/trusted.gpg.d/cran_ubuntu_key.asc
+            sudo add-apt-repository -y "deb https://cloud.r-project.org/bin/linux/ubuntu $(lsb_release -cs)-cran40/"
+            sudo add-apt-repository -y ppa:c2d4u.team/c2d4u4.0+
+            sudo apt update && sudo apt install -y ccache pandoc libgit2-dev libharfbuzz-dev \
+            libfribidi-dev libprotoc-dev libprotobuf-dev libv8-dev librsvg2-dev libmpfr-dev \
+            libnlopt-dev libgdal-dev libproj-dev r-base r-cran-devtools r-cran-tensorflow \
+            python3-pip python3.10-venv
+            R -q -e 'if (!dir.exists("~/R/library")) dir.create("~/R/library", recursive = TRUE)'
+            echo -e 'options(Ncpus = 4, repos = structure(c(CRAN = "https://cloud.r-project.org/")))' > $HOME/.Rprofile
+            export R_HOME=`R RHOME`
+            echo -e ".libPaths(c('$HOME/R/library', '/usr/lib/R/site-library', '/usr/lib/R/library'))" | \
+            sudo tee -a $R_HOME/etc/Rprofile.site
+            R -q -e 'remotes::install_github("Displayr/flipDevTools")'
+            python3.10 -m venv ~/.virtualenvs/r-tensorflow
+            source ~/.virtualenvs/r-tensorflow/bin/activate
+            python -m pip install --upgrade pip
+            python -m pip install tensorflow keras
+            echo "$(which python)"
+            echo 'RETICULATE_PYTHON="~/.virtualenvs/r-tensorflow/bin/python"' | \
+            sudo tee -a $R_HOME/etc/Renviron.site
+            R -q -e 'remotes::install_github("cran/survey@4.0")'
```

Comment:
Can add `cran/survey@4.0` to `Remotes` in the `flipData` DESCRIPTION file and remove this line